.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -7,14 +7,20 @@ assignees: ''
 
 ---
 
-**Describe the bug**
+### Relevant Discussion
+
+Non-trivial features require a lot of discussion to find the best implementation in the target languages.
+This discussion should take place in [discussion tab](https://github.com/pyccel/pyccel/discussions/categories/q-a).
+Please link to the discussion where the proposed feature was discussed.
```

Comment:
In this section I count 7 occurrences of the word "discussion" 😄 Maybe we can rephrase and use some synonyms?

Reply:
Fixed 789fb4e3